es_ES/faq.md

```diff
@@ -0,0 +1,625 @@
+
+Además, tenemos la oportunidad de trabajar estrechamente con la comunidad Dart,
+que está invirtiendo activamente recursos para mejorar Dart para usar en Flutter. por
+ejemplo, cuando adoptamos Dart, el lenguaje no tenía una AOT toolchain para producir binarios nativos, que es fundamental para lograr predecible, de alto rendimiento, pero ahora el lenguaje lo hace porque el equipo de Dart lo construyó para Flutter. Del mismo modo, la Dart VM se ha optimizado previamente para rendimiento pero el equipo ahora está optimizando la VM para la latencia, que es más
+importante para la carga de trabajo de Flutter.
+
+Dart tiene una alta puntuación para nosotros en los siguientes criterios principales:
+
+* _Productividad del desarrollador_. Una de las principales propuestas de valor de Flutter es que ahorra recursos de ingeniería al permitir que los desarrolladores creen aplicaciones tanto para iOS como para Android con la misma base de código. Usar un lenguaje altamente productivo acelera aún más a los desarrolladores y hace que Flutter sea más atractivo. Esto fue muy importante tanto para nuestro equipo de framework como para nuestros desarrolladores. La mayoría de Flutter está construido en el mismo idioma que le damos a nuestros usuarios, por lo que necesitamos seguir siendo productivos en las líneas de código de 100k, sin sacrificar accesibilidad o legibilidad del framework y widgets para nuestros desarrolladores.
+
+* _Orientación a objetos_. Para Flutter, queremos un lenguaje que se adapte a  problema de dominio de Flutter: crear experiencias visuales de usuario. La industria tiene varias décadas de experiencia construyendo framework de interfaz de usuario en lenguajes orientados a objetos. Si bien podríamos usar un lenguaje no orientado a objetos, esto significaría reinventar la rueda para resolver varios problemas difíciles. Además, la gran mayoría de los desarrolladores tienen experiencia con desarrollo orientado a objetos, por lo que es más fácil aprender a desarrollar con Flutter.
+
+* _Predictable, alto rendimiento_. Con Flutter, queremos potenciar a los desarrolladores para crear experiencias de usuario rápidas y fluidas. Para lograr eso, necesitamos ser capaz de ejecutar una cantidad significativa de código de desarrollador final durante cada marco de animación. Eso significa que necesitamos un lenguaje que ofrezca altos rendimiento y rendimiento predecible, sin pausas periódicas que causarían marcos caídos.
+
+* _Fast allocation_. El framework de Flutter usa un flujo de estilo funcional eso depende mucho de la memoria manejando eficientemente asignaciones pequeñas y de corta duración. Este estilo fue desarrollado en idiomas con esta propiedad y lo hace no funciona de manera eficiente en idiomas que carecen de esta facilidad.

```

Comment:
The Flutter framework uses a functional-style flow that depends heavily on the underlying memory allocator efficiently handling small, short-lived allocations
El framework Flutter usa un flujo de estilo funcional que depende en gran medida del asignador de memoria subyacente manejando eficientemente asignaciones pequeñas y de corta duración